src/Gen/Array.elm

```diff
@@ -1,7 +1,7 @@
 module Gen.Array exposing (annotation_, append, call_, empty, filter, foldl, foldr, fromList, get, indexedMap, initialize, isEmpty, length, map, moduleName_, push, repeat, set, slice, toIndexedList, toList, values_)
```

Comment:
Btw I usually put the `Gen/` directory in `.gitignore` because these modules aren't a source of truth and can be generated locally.
Is there a reason not to do that?

Reply:
Yupp: elm-generic-dict needs (some) of those `Gen` files for its internal use. I should probably `.gitignore` the others

Reply:
https://github.com/miniBill/elm-generic-dict/blob/main/src/GenericDict.elm#L30-L34

Reply:
Thanks for the idea, fixed in 86383ff

Reply:
Yes of course. What I meant was the fact that for example `Gen.FastDict` doesn't need to be stored in git because it can be reconstructed perfectly be `elm-codegen`. Same reason you don't commit your `VerifyExamples/` directory etc.